examples/quine.html

```diff
@@ -3,10 +3,13 @@
 <head>
 <meta charset="utf-8" />
 <title>Making Quines Prettier</title>
-<!-- The defer is not necessary for autoloading, but is necessary for the
-     script at the bottom to work as a Quine. -->
-<script src="https://cdn.rawgit.com/google/code-prettify/master/loader/run_prettify.js?autoload=true&amp;skin=sunburst&amp;lang=css" defer="defer"></script>
-<style>.operative { font-weight: bold; border:1px solid yellow }</style>
+<!-- The defer attribute is not necessary for autoloading, but is necessary
+     for the script at the bottom to work as a Quine. -->
+<script src="https://cdn.rawgit.com/google/code-prettify/master/loader/run_prettify.js?autoload=true&amp;skin=sunburst&amp;lang=css" defer></script>
```

Comment:
Agreed, we talked about this before too.

The current tags named [`yyyy-mm-dd`](https://github.com/google/code-prettify/tags) were ported from SVN and not very recent, the idea is to switch to `v1.2.3` style for future releases.

We yet have to settle on a process for publishing releases (we talked about publishing packages to npm and bower as well)..

In any case I'll change `cdn.rawgit.com` to `rawgit.com` for demos examples and tests, which doesn't have an aggressive caching. That would be more appropriate right?